packages/fontaine/src/index.ts

```
@@ -1,7 +1,72 @@
+ */
+export const generateFontMetrics = (options) => {
+    return generateMetricsCSS(options);
+};
+
+/**
+ * Calculates and generates CSS fallback rules for a given font name.
+ * @param {string} fontName The name of the primary font.
+ * @param {FallbackOptions} [options] Options governing the fallback process.
+ * @returns {string} The CSS fallback rules.
+ */
+export const generateFallbackCSS = (fontName, options) => {
```

ESLint

[error] 'generateFallbackCSS' is already defined.

(ts/no-redeclare)